zuul.d/project-templates.yaml

```diff
@@ -720,6 +720,7 @@
         - ansible-test-units-kubernetes-core-python37
         - ansible-test-integration-kubernetes-core
         - ansible-test-molecule-kubernetes-core
+        - ansible-test-lint-kubernetes-core
```

Comment:
You can just use the existing `ansible-tox-linters` template.

Reply:
I did not know that it was existing, thanks